modin/pandas/test/dataframe/test_join_sort.py

```diff
@@ -776,7 +776,7 @@ def test_sort_values_with_string_index():
 
 
 @pytest.mark.skipif(
-    Engine.get() not in ["Ray", "Dask", "Unidist"],
+    Engine.get() not in ("Ray", "Unidist", "Dask"),
```

Comment:
just curious, does this make a big difference? :)

Reply:
No, but this is unification with other parts of the code.